src/app/components/main/main.js

```diff
@@ -32,15 +30,23 @@ app.component('main', {
 
       this.$rootScope.prelogged = true;
 
-      this.update()
+      this.update(attempts)
         .then(() => {
           this.$rootScope.prelogged = false;
           this.$rootScope.logged = true;
-          this.checkIfIsDelegate();
+          if (this.$timeout) {
```

Comment:
I don't understand the purpose of this `if` statement, but it looks weird. Maybe add some comment.